sdk/tools/azure-openrewrite/src/test/resources/migrationExamples/azure-core/sample10/v2/HttpPostMultipartExample.java

```diff
@@ -0,0 +1,31 @@
+import io.clientcore.core.http.client.HttpClient;
+import io.clientcore.core.http.client.JdkHttpClientBuilder;
+import io.clientcore.core.http.models.HttpHeaderName;
+import io.clientcore.core.http.models.HttpHeaders;
+import io.clientcore.core.http.models.HttpMethod;
+import io.clientcore.core.http.models.HttpRequest;
+import io.clientcore.core.http.models.Response;
+import io.clientcore.core.models.binarydata.BinaryData;
+import java.io.IOException;
+
+public class HttpPostMultipartExample {
+    public static void main(String... args) {
+        HttpClient client = new JdkHttpClientBuilder().build();
+        HttpHeaders headers = new HttpHeaders();
+        headers.add(HttpHeaderName.CONTENT_TYPE, "multipart/form-data");
```

Comment:
Should follow the same pattern as before - `HttpHeaders headers = new HttpHeaders().set("Content-Type", "multipart/form-data");`